app/views/rapidfire/answers/_checkbox.html.erb

```diff
@@ -4,7 +4,7 @@
 <%= f.fields_for :answer_text do |af| %>
   <%- answer.question.options.each_with_index do |option, index| %>
     <%= af.label index do %>
-      <%= af.check_box index, { checked: checkbox_checked?(answer, option) }, option %>
+      <%= check_box_tag "attempt[#{answer.question.id}][answer_text][#{index}]", option, checkbox_checked?(answer, option) %>
```

Comment:
Hardcoding name might not be a good idea. Any other way we can generate the name?

Reply:
I suppose you could replace "attempt[" with something like "#{f.object.class.model_name.param_key}[" but I do not know how to use the Rails builder for generating the checkbox without it generating the `0` answer too, which breaks other things, as noted in the commit message. 

I also would assume you could get "answer_text" from `af` somehow, but since it is already in there in the `fields_for` I don't see what is to be gained for that one.

Reply:
Merging this one, I've one more PR which adds test for this, I'll fix this part in that PR.